### PR TITLE
Add hirak/prestissimo to speed up composer install

### PR DESCRIPTION
**Pull request by WyriHaximus**
Done this in my own images and it significantly speeds up composer
installs/requires/updates.

# Usabilla PHP Docker Template

Reviewers: @usabilla/oss-docker

## Type

Please specify the type of changes being proposed:

| Q                 | A
| ----------------- | -----
| Documentation?    | no
| Dockerfile change?| yes
| Build feature?    | no
| Apply CVE Patch?  | no
| Remove CVE Patch? | no

### CODE REVIEW

**Comment by WyriHaximus** [comment]
Sure, want me to PR moving composer to `-dev` images only first then? Doing the same for my own images: https://github.com/WyriHaximusNet/docker-php/blob/master/Dockerfile-zts#L100-L126

File: Dockerfile-cli [hunk 1]
```diff
@@ -39,6 +39,8 @@ RUN install-shush && rm -rf /usr/local/bin/install-shush
 COPY src/php/utils/install-composer /usr/local/bin/
 RUN install-composer && rm -rf /usr/local/bin/install-composer
 
+RUN composer global require hirak/prestissimo --ansi --no-progress
```
[the remark this answers is not in the document]
Comment (WyriHaximus) [reply Dockerfile-cli]: Good one thanks, wouldn't do a lot filesize wise but all little bits help keeping docker images size down :+1:

[the remark this answers is not in the document]
Comment (WyriHaximus) [reply Dockerfile-cli]: Just did this on my own images and saved about 10MB to 20MB :tada:

**Comment by rdohms** [comment]
As we talked today at the office about the risks of adding an non-core extension like this t our images, shall we close this PR in favor of one that documents how to do it and leave it at the will of end users?

**Comment by WyriHaximus** [comment]
@rdohms yes :+1:

**Pull request closed without merging**